packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,13 +1,36 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import { PaperPlane } from '@strapi/icons';
+
+import { pluginId } from './pluginId';
+
+import type { Plugin } from '@strapi/types';
 
 // eslint-disable-next-line import/no-default-export
-export default {
-  register() {
+const admin: Plugin.Config.AdminInput = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register(app: any) {
     if (window.strapi.features.isEnabled('cms-content-releases')) {
       // EE Code would be here
       // For testing if the plugin is working, add a console.log
+      app.addMenuLink({
+        to: `/plugins/${pluginId}`,
+        icon: PaperPlane,
+        intlLabel: {
+          id: `${pluginId}.plugin.name`,
+          defaultMessage: 'Releases',
+        },
+        async Component() {
+          const { Releases } = await import(
+            /* webpackChunkName: "content-type-builder" */ './pages/Releases'
+          );
+          return Releases;
+        },
+        permissions: [], // array of permissions (object), allow a user to access a plugin depending on its permissions
+      });
     }
   },
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  bootstrap() {},
```

Comment:
I know we have this in some plugins, but if you look the `color-picker` plugin I think is not necessary, so lets remove it and add it then if we have something to put inside the bootstrap

Reply:
Same reason, I used the type definition we used for the admin in the email plugin https://github.com/strapi/strapi/blob/9e14912c58fadb39fda6d8e78015c27eed9a960c/packages/core/email/admin/src/index.ts#L14 and if you use this type you need also to add the bootstrap

Reply:
ok I changed bootstrap to be optional in the Config type